tgui/packages/tgui/interfaces/MessageMonitor.tsx

```diff
@@ -0,0 +1,385 @@
+import { Window } from '../layouts';
+
+enum Screen {
+  Main,
+  MessageLogs,
+  RequestLogs,
+  Hacked,
+}
+
+enum ServerStatus {
+  Offline,
+  Online,
+}
+
+type Data = {
+  screen: Screen;
+  status: BooleanLike;
+  server_status: number;
+  auth: number;
+  password: string;
+  is_malf: BooleanLike;
+  error_message: string;
+  success_message: string;
+  notice_message: string;
+  requests: Request[];
+  messages: Message[];
+};
+
+type Request = {
+  ref: string;
+  message: string;
+  stamp: string;
+  send_dpt: string;
+  id_auth: string;
+};
+
+type Message = {
+  ref: string;
+  message: string;
+  sender: string;
+  recipient: string;
+};
+
+const RequestLogsScreen = (props, context) => {
+  const { act, data } = useBackend<Data>(context);
+  const { requests = [] } = data;
+  return (
+    <Stack fill vertical>
+      <Stack.Item grow>
+        <Section
+          fill
+          scrollable
+          title="Requests"
+          buttons={
+            <Button
+              content="Main Menu"
+              icon="home"
+              onClick={() => act('return_home')}
+            />
+          }>
+          <Table>
+            <Table.Row header>
+              <Table.Cell>Delete</Table.Cell>
+              <Table.Cell>Message</Table.Cell>
+              <Table.Cell>Stamp</Table.Cell>
+              <Table.Cell>Departament</Table.Cell>
+              <Table.Cell>Authentication</Table.Cell>
+            </Table.Row>
+            {requests?.map((request) => (
+              <tr key={request.ref} className="Table__row candystripe">
+                <td>
+                  <Button
+                    icon="trash"
+                    color="red"
+                    onClick={() => act('delete_request', { ref: request.ref })}
+                  />
+                </td>
+                <td>{request.message}</td>
+                <td>{request.stamp}</td>
+                <td>{request.send_dpt}</td>
+                <td>{request.id_auth}</td>
+              </tr>
+            ))}
+          </Table>
+        </Section>
+      </Stack.Item>
+    </Stack>
+  );
+};
+
+const MessageLogsScreen = (props, context) => {
+  const { act, data } = useBackend<Data>(context);
+  const { messages = [] } = data;
+  return (
+    <Stack fill vertical>
+      <Stack.Item grow>
+        <Section
+          fill
+          scrollable
+          title="Stored Messages"
+          buttons={
+            <Button
+              content="Main Menu"
+              icon="home"
+              onClick={() => act('return_home')}
+            />
+          }>
+          <Table>
+            <Table.Row header>
+              <Table.Cell>Delete</Table.Cell>
+              <Table.Cell>Sender</Table.Cell>
+              <Table.Cell>Recipient</Table.Cell>
+              <Table.Cell>Message</Table.Cell>
+            </Table.Row>
+            {messages?.map((message) => (
+              <tr key={message.ref} className="Table__row candystripe">
```

Comment:
this is probably okay, but you should have been able to use the component,
```<Table.Row key={message.ref} className="candystripe">```
same for below with Table.cell